### ISSUE TITLE
dynamodb.Query can't get ConsumedCapacity

### ISSUE THREAD

**Issue by nekakoshi** [localstack/localstack#1108]
Hi contributor.

I want get `ConsumedCapacity` in dynamodb when execute `dynamodb.Query`.

I think unsupported return `ConsumedCapacity` in [dynamodb.Query](https://github.com/localstack/localstack/blob/master/localstack/services/dynamodb/dynamodb_listener.py#L185) .

I want to be supported like to `'%s.GetItem'` and/or another solution. :) 

cf: in my localstrage

```
@@ -182,6 +188,25 @@ class ProxyListenerDynamoDB(ProxyListener):
             if 'StreamSpecification' in data:
                 create_dynamodb_stream(data)
             return
+        elif action == '%s.Query' % ACTION_PREFIX:
+            print('%s.Query')
+            print(response.status_code)
+            if response.status_code == 200:
+                content = json.loads(to_str(response.content))
+                print(content)
+                # make sure we append 'ConsumedCapacity', which is properly
+                # returned by dynalite, but not by AWS's DynamoDBLocal
+                print('ConsumedCapacity')
+                print(content)
+                print(data.get('ReturnConsumedCapacity'))
+                if 'ConsumedCapacity' not in content and data.get('ReturnConsumedCapacity') in ('TOTAL', 'INDEXES'):
+                    print('hogehoge')
+                    content['ConsumedCapacity'] = {
+                        'CapacityUnits': 0.5,  # TODO hardcoded
+                        'TableName': data['TableName']
+                    }
+                    response._content = json.dumps(content)
+                    fix_headers_for_updated_response(response)
         else:
             # nothing to do
             return
```

**Comment by whummer** [comment]
Thanks for reporting @nekakoshi . Do you think you could clone this repository, and create a pull request with your changes? 

Also, let's try to avoid code duplication if possible. Since the identical logic already exists, I think we could simply change this line in `dynamodb_listener.py`:
```
elif action == '%s.GetItem' % ACTION_PREFIX:
```
... to this line:
```
elif action in ('%s.GetItem' % ACTION_PREFIX, '%s.Query' % ACTION_PREFIX):
```

What do you think?

**Comment by nekakoshi** [comment]
@whummer
Thanks your Idea.

I want to try the following method.
```
elif action in ('%s.GetItem' % ACTION_PREFIX, '%s.Query' % ACTION_PREFIX):
```

If you plan to extend `.Query`, 
I think the common method is good.

**Comment by whummer** [comment]
Thanks again - this should be fixed in #2071 . Please report here if the problem persists. Thanks
